Backend/BikesRentalServer.Tests/StationsService/BlockStation.cs

```diff
@@ -0,0 +1,95 @@
+﻿using BikesRentalServer.Authorization;
+using BikesRentalServer.DataAccess;
+using BikesRentalServer.Dtos.Requests;
+using BikesRentalServer.Models;
+using BikesRentalServer.Services;
+using BikesRentalServer.Tests.Mock;
+using FluentAssertions;
+using Xunit;
+
+namespace BikesRentalServer.Tests.StationsService
+{
+    public class BlockStation
+    {
+        private readonly DatabaseContext _dbContext;
+        private readonly Services.StationsService _stationsService;
+
+        public BlockStation()
+        {
+            _dbContext = MockedDbFactory.GetContext();
+            _stationsService = new Services.StationsService(_dbContext, new UserContext());
+        }
+
+        [Fact]
+        public void BlockStationShouldReturnBlockedStation()
+        {
+            var station = _dbContext.Stations.Add(new Station
+            {
+                Name = "DS Ustronie",
+                Status = BikeStationStatus.Working,
+            })
```

Comment:
Ech, znowu... umieść ten obiekt i `.Entity` na jednym poziomie, żeby to jakoś wyglądało